src/SIL.XForge.Scripture/ClientApp/src/app/join/join.component.html

```diff
@@ -1 +1,18 @@
-<div></div>
+<ng-container *transloco="let t; read: 'join'">
+  <ng-container *ngIf="!isLoading">
+    <h1>{{ t("join_a_project") }}</h1>
+    <p [innerHTML]="inviteText"></p>
+    <app-notice *ngIf="!isOnline" type="error" icon="cloud_off">{{ t("please_connect_to_use_link") }}</app-notice>
+    <form>
+      <mat-form-field appearance="fill">
+        <mat-label>{{ t("your_name") }}</mat-label>
+        <input matInput [formControl]="name" />
+      </mat-form-field>
+      <button mat-flat-button color="primary" type="submit" (click)="joinProject()" [disabled]="!isFormEnabled">
+        {{ t("join") }}
+      </button>
+    </form>
+    <a (click)="logIn()" [innerHtml]="i18nService.translateAndInsertTags('join.login_existing_account')"></a>
```

Comment:
Is it possible to make "Log in" a proper hyperlink instead of underlined text? I found the lack of normal hyperlink hover effects confusing, and made me think it was not a real link.

Reply:
Hmmm the log in link still doesn't feel right to me. This isn't my area of expertise (i.e. this is not a blocker), but would something like this work:

![image](https://user-images.githubusercontent.com/8665431/233861361-65b663c4-b83a-4ca9-b3fa-7042c3a9f544.png)

If it doesn't feel better to you, just tell me no :-)